coops-ui/src/App.vue

```diff
@@ -54,9 +54,6 @@ import AccountAuthorizationDialog from '@/components/Dashboard/AccountAuthorizat
 // Mixins
 import { DateMixin, CommonMixin } from '@/mixins'
 
-// Enums
-import { EntityTypes } from '@/enums'
```

Comment:
Did i import the enum but not use it?  This was probably a result of me importing the enum for use when hardcoding the `setEntityType` during testing. Good job on the clean up, thank you.

Reply:
yw